crates/matrix-sdk/src/authentication/oidc/mod.rs

```diff
@@ -644,6 +644,46 @@ impl Oidc {
         Ok(Some(url))
     }
 
+    /// Get the account management URL where the user can manage their
+    /// identity-related settings.
+    ///
+    /// # Arguments
+    ///
+    /// * `action` - An optional action that wants to be performed by the user
+    ///   when they open the URL. The list of supported actions by the account
+    ///   management URL can be found in the [`VerifiedProviderMetadata`], or
+    ///   directly with [`Oidc::account_management_actions_supported()`].
+    ///
+    /// Returns `Ok(None)` if the URL was not found. Returns an error if the
+    /// request to get the provider metadata fails or the URL could not be
+    /// parsed.
+    ///
+    /// This method will cache the URL for a while, if the cache is not
+    /// populated it will internally call
+    /// [`Oidc::fetch_account_management_url()`] and cache the resulting URI
```

Comment:
how nitpicky of me, but there's an inconsistency between `URI` and `URL` in this paragraph :nerd_face:

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/4663/commits/768ec36376896384407fcfa42d968d5344402075